setup.py

```diff
@@ -34,7 +34,7 @@
     python_requires='>=2.7',
     install_requires=[
         'pandas<0.20',
```

Comment:
Is this a hard limit? And what is this hard limit for? I'd recommend loosening this pin a little if possible (with an environment marker if this project also needs to run on Python 3.4, let me know if this is the case and I can update this list of requires.)